app/routers/features.py

```diff
@@ -0,0 +1,147 @@
+from fastapi import APIRouter, Request, Depends
+
+from app.dependencies import get_db, SessionLocal
+from app.database.models import UserFeature, Feature
+from app.internal.features import (
+    create_association,
+    is_association_exist_in_db,
+    is_feature_exist_in_disabled,
+    is_feature_exist_in_enabled,
+    get_user_disabled_features,
+    get_user_enabled_features
+)
+
+router = APIRouter(
+    prefix="/features",
+    tags=["event"],
+    responses={404: {"description": "Not found"}},
+)
+
+
+@router.get('/')
+async def index(request: Request, session: SessionLocal = Depends(get_db)):
+    features = session.query(Feature).all()
+    return features
+
+
+@router.post('/add-feature')
```

Comment:
`/feature/add`

Reply:
@yammesicka: wouldn't @router.post('/') or @router.put work here?
So:
.get('/') - get list
.post('/') (or .put('/')) - add
.delete('/') - delete

Reply:
@Gonzom  Why `/feature/add` is not good enough?
There is a lot of post requests and get requests as well and in my opinion explicit is better then implicit :)